Assets/Scripts/AngleRotation.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AngleRotation : SteeringMode {
+    const string NAME = "Angle rotation";
+    int[] angles = {30, 45, 60, 90};
+    bool rotating = false;
+    int chosenAngle;
+
+    public AngleRotation() {
+        System.Random rnd = new System.Random();
+        chosenAngle = angles[rnd.Next(0, angles.Length)];
+    }
+
+    /**
+        Move the board towards the right
+    */
+    public override void moveRight(GameObject board, Action<IEnumerator> startMovement) {
+        if(!rotating) {
+            startMovement(Rotate(new Vector3(0, chosenAngle, 0), board));
+        }
+    }
+
+    /**
+        Move the board towards the left
+    */
+    public override void moveLeft(GameObject board, Action<IEnumerator> startMovement){
+        if(!rotating) {
+            startMovement(Rotate(new Vector3(0, -chosenAngle, 0), board));
+        }
+    }
+
+    /**
+        Get name of steering mode
+    */
+    public override string getName() {
+        return NAME;
+    }
+    
+    /**
+        Rotate a set amount of degrees over time
+    */
+    private IEnumerator Rotate(Vector3 angles, GameObject board) {
+        float duration = 1.0f;
```

Comment:
When rotating 90 degrees, the time feels okay, but when rotating less it feels a bit to slow. Do we want the duration to be shorter when its rotating less?

Reply:
Should be better now ^^